lib/potassium/cli.rb

```diff
@@ -1,3 +1,6 @@
+# system('gem list | grep gli')
+puts 'cli!!!'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.